src/Avalonia.Headless/AvaloniaHeadlessPlatform.cs

```diff
@@ -52,10 +52,7 @@ class HeadlessWindowingPlatform : IWindowingPlatform
 
             public IPopupImpl CreatePopup() => new HeadlessWindowImpl(true);
 
-            public ITrayIconImpl CreateTrayIcon()
-            {
-                throw new NotImplementedException();
-            }
+            public ITrayIconImpl CreateTrayIcon() => return null;
```

Comment:
this code is not valid and breaks the pipeline.

Reply:
fixed.